README.md

```diff
@@ -113,27 +113,56 @@ pip install -r "requirements.txt"
 pip install git+https://github.com/buzzn/discovergy.git
 ```
 
-Create an endpoint in your webserver, for example for Apache, create a new site:
+Now create a systemd service `mybuzzn-backend.service`, which starts the socketio server we use to run
+the app in:
+```ini
+[Unit]
+Description=Run mybuzzn-backend as an flask socketio instance.
+
+[Service]
+WorkingDirectory=/var/www/mybuzzn-backend.buzzn.net/mybuzznbackend
+ExecStart=/var/www/mybuzzn-backend.buzzn.net/mybuzznbackend/env/bin/python mybuzzn-backend.py
+After=network.target
+User=mybuzznbackend
+Group=mybuzznbackend
+
+[Install]
+WantedBy=multi-user.target
 ```
-<VirtualHost *:80>
-  ServerName your-domain.net
-  ErrorLog /path/to/error/log
-  CustomLog /path/to/access.log combined
+Either store it on `/etc/systemd/system/` or create there a link pointing to it.
```

Comment:
"Either store it on `/etc/systemd/system/` or create a link there pointing to it."